src/WebAssert.php

```diff
@@ -46,12 +47,17 @@ public function __construct(Session $session)
      *
      * @throws ExpectationException
      */
-    public function addressEquals($page)
+    public function addressEquals($page, $timeout = 10)
     {
-        $expected = $this->cleanUrl($page);
-        $actual = $this->getCurrentUrlPath();
+        $cleanedUrl = $this->cleanUrl($page);
+        $actual = null;
 
-        $this->assert($actual === $expected, sprintf('Current page is "%s", but "%s" expected.', $actual, $expected));
+        $this->assert(
+            $this->session->getPage()->waitFor($timeout, function () use ($cleanedUrl, &$actual) {
```

Comment:
@lenybernard Do you think that we should keep the `waitFor()` for the address? I'm pretty sure that the address of a page is not supposed to change once a page if loaded (if Mink wait for redirections, I didn't check that).